cranelift/codegen/meta/src/isa/mod.rs

```diff
@@ -7,6 +7,7 @@ mod arm32;
 mod arm64;
 mod riscv;
 mod x86;
+mod spirv;
```

Comment:
@cfallin Is it still necessary to specify backends in `codegen-meta` when using the new framework?

Reply:
It should be only useful to require a new-backend style meta build, that is, build the instructions and `expand` group. So the module is not strictly required here, the `Isa` variant below is sufficient, and should be handled in the second loop in `codegen/meta/src/lib.rs`, probably with an empty match arm (as aarch64) if there are no spirv specific settings. Also the `codegen/build.rs` script should mimic what's done for x64.